src/graphql/community-helps.ts

```diff
@@ -24,3 +24,18 @@ export const COMMUNITY_HELP = `
     }
   }
 `
+
+export const HELPFUL_THREADS = `
+  query HelpfulThreads {
+    CommunityHelps(limit: 0, where: { helpful: { equals: true} }) {
```

Comment:
Shouldn't we be querying on whether or not there are `relatedDocs`. The `helpful` property might be good if we are indexing on it so that the db can improve it's own performance, but this alone isn't even the best way to find these. Having both helpful and related could be bad too since we are now using human input to do this.